### DISCUSSION TITLE
[1960270]Lootun {本体}

### DISCUSSION THREAD

**Discussion by rqelove510** [pjy612/SteamManifestCache#655] [Share]
### Steam 游戏ID

1960270

### 密钥列表

```hash
1960271;b5d5f1bd784dea43a9d929ebcc65b50f78f389fc736a77bc6c6a3be00cda669a
```

### 清单附件

[1960271_2369279539943399502.zip](https://github.com/pjy612/SteamManifestCache/files/15345698/1960271_2369279539943399502.zip)

### 备注

Code:621DB7D84708F4B7
Email:hencq@vip.qq.com

**Comment by pjy612** [comment]
* 游戏id 错了 #26
* 清单更新

**Reply by rqelove510** [reply]
已更新ID和清单，标题ID还是错的，编辑不了了，以内容为准。请求加入内测！

**Reply by pjy612** [reply]
* 右上应该有 edit 按钮
* 他又更新了...

**Reply by rqelove510** [reply]
1.0.2已更新清单，不易：（
